README-PYPI.md

```diff
@@ -24,7 +24,7 @@ If you just want the latest version this should be sufficient:
 pip install vss-tools
 ```
 
-When installed tools like `vspec2json` shall be available on your path.
+When installed the cli tool `vspec` should be available in your `PATH`.
```

Comment:
As we have changed how we package top level files to pypi I think this file needs a little update. At lines 7-17 we should either list binary tool or remove the list of tools altogether (and possibly instead add a link to this repo).

And as we do no exclude binary tool we should maybe somewhere add a little note that even if binary tool is there, they still need to create the *.so file to use?

Reply:
Done